.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,72 @@
-### Prerequisites
+## Prerequisites
 
-- [ ] I have read and understood the [contributing guide][CONTRIBUTING.md].
-- [ ] The commit message follows the [conventional commits][cc] guidelines.
-- [ ] Tests for the changes have been added (for bug fixes / features).
-- [ ] Docs have been added/updated (for bug fixes / features).
+- [ ] I have read and understood the [Contributing Guide][contributing].
+- [ ] My commit messages follow the [Conventional Commits][cc] guidelines and our [commitlint rules and standards][commitlint].
+- [ ] Relevant tests have been added or updated (for bug fixes or features).
+- [ ] Documentations have been added or updated (for bug fixes or features).
 
-### Description
+## Description
 
-<!---
+<!--
 
-Tips:
+**Description of Changes:**
 
-If you're not comfortable with working with Git,
-we're working a guide (https://ohmyposh.dev/docs/contributing/git) to help you out.
-Oh My Posh advises GitKraken (https://www.gitkraken.com/invite/nQmDPR9D) as your preferred cross platform Git GUI power tool.
+Please provide a clear and concise description of the changes introduced by this pull request. You should explain the following:
+
+1. **Summary:** Briefly describe the main change or addition in this PR. What is the overall purpose of your changes?
+2. **Problem:** Specify the issue or problem this PR addresses. Include any relevant context or background to explain why this change is necessary.
+3. **Solution:** Explain how your changes resolve the issue or add the new feature. Provide key implementation details to clarify how the solution works.
+4. **Impact:** Note any potential impacts or side effects of your changes. Mention if additional testing is needed or if there are any known limitations.
+5. **Issue Closure:** Indicate which issues this PR resolves or closes.
+
+**Example:**
+"This PR introduces a new feature for X by implementing Y that will optimize the query logic, which reduces execution time and prevents the error. This change addresses the issue reported in #123.
+
+Closes #123"
+
+Please ensure that each section is thoroughly explained to assist reviewers in understanding your changes.
 
 -->
 
-[CONTRIBUTING.md]: https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/CONTRIBUTING.md
+---
+
+<br>
```

Comment:
why are the linebreaks here?

Reply:
The line breaks are used to emphasize the main PR description section, making it easier to focus on providing a clear and concise overview of the changes.